src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -394,60 +394,6 @@
 
       it { expect { get_binaries }.to raise_error(ActiveRecord::RecordNotFound) }
     end
-
-    context 'with build results and no binaries' do
-      render_views
-
-      before do
-        allow(Backend::Api::BuildResults::Status).to receive(:result_swiss_knife).and_return(fake_build_results_without_binaries)
-        get :binaries, params: { package: source_package, project: source_project, repository: repo_for_source_project.name }
-      end
-
-      it { expect(response).to have_http_status(:success) }
-      it { expect(response.body).to match(/No built binaries/) }
-    end
-
-    context 'with build results and binaries' do
```

Comment:
As we remove this tests, we could also remove its cassette and the complete subdirectory: `spec/cassettes/Webui_PackageController/GET_binaries/`.